python/ray/ml/tests/test_torch_predictor.py

```diff
@@ -45,6 +45,20 @@ def test_predict():
     assert predictions.to_numpy().flatten().tolist() == [4, 8, 12]
 
 
+def test_predict_array_output():
+    """Tests if predictor works if model outputs an array instead of single value."""
+
+    predictor = TorchPredictor(model=model, preprocessor=preprocessor)
```

Comment:
Just for my own understanding, why do we need to use a preprocessor here?

Reply:
I believe it is to check that its applied, though that could be covered by other tests and not this one specifically

Reply:
In this case I'd vote to leave it out of this test. We should ideally only test one thing

Reply:
Sounds good,  I updated the tests!